src/main.c

```diff
@@ -63,10 +63,10 @@ enum app_state {
  * Passive mode: Sensor data and GPS position is acquired when movement is
  *		 detected, or after the configured movement timeout occurs.
  */
-enum app_sub_state {
-	APP_SUB_STATE_ACTIVE_MODE,
-	APP_SUB_STATE_PASSIVE_MODE,
-} app_sub_state;
+static enum sub_state_type {
+	STATE_ACTIVE_MODE,
```

Comment:
Prefix with `SUB_STATE` to align with variable name ?

Reply:
Yep.